### ISSUE TITLE
(Failure nth)

### ISSUE THREAD

**Issue by talex5** [lpw25/prof_alloc#15]
Running `prof_spacetime view -e /mnt/datakit -p /mnt/spacetime-1.p` and pressing Return on a page with no options caused it to crash with:

```
Processing series...done
prof-spacetime: internal error, uncaught exception:
                (Failure nth)
```

**Comment by lpw25** [comment]
(Side-note: you don't need `-e` if you've already processed the file)

**Comment by lpw25** [comment]
This should be fixed in 0.2.0.
